src/GraphQL/Instrumentation/FieldMiddlewareBuilderExtensions.cs

```diff
@@ -10,7 +10,7 @@ namespace GraphQL.Instrumentation
     public static class FieldMiddlewareBuilderExtensions
     {
         /// <summary>
-        /// Adds middleware to the list of delegates that will be applied to the schema when invoking <see cref="IFieldMiddlewareBuilder.ApplyTo(ISchema)"/>.
+        /// Adds middleware to the list of delegates that will be applied to the schema when invoking <see cref="SchemaTypes.ApplyMiddleware(IFieldMiddlewareBuilder, ISchema)"/>.
```

Comment:
>  will be applied to the **schema**

I would like to change the emphasis in all comments of this kind. Let's say delegates apply to **all field resolvers, not schema**. This is more accurate and understandable.